index.rst

```diff
@@ -1,61 +1,109 @@
-..
-  Technote content.
+:tocdepth: 1
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+.. sectnum::
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Abstract
+========
 
-  Use the following syntax for sections:
+Proprietary Rubin Observatory astronomical data must be transferred from the summit facility where the data is captured to the USDF (United States Data Facility) for processing.
+This document proposes a data transfer implementation strategy to maintain confidentiality of the data in transit.
+It addresses the concerns identified in DMTN-108_.
 
-  Sections
-  ========
+.. _DMTN-108: https://dmtn-108.lsst.io/
 
-  and
+Problem statement
+=================
 
-  Subsections
-  -----------
+Data captured by the camera attached to the Charles Simonyi Survey Telescope on Cerro Pachón in Chile will be transferred from the summit facility (via dedicated private fiber) to the :abbr:`USDF (United States Data Facility)` for processing.
```

Comment:
"Dedicated private fiber" only applies to some segments of the full path to the USDF.  Perhaps leave out the parenthetical altogether?

Reply:
Oh, whoops, that was left over from the summit to base discussion. Good catch.